src/pages/Home/home.js

```diff
@@ -1,7 +1,156 @@
+      );
+      if (subject === "horror") {
+        this.setState({ horror: data.items });
+      } else if (subject === "romance") {
+        this.setState({ romance: data.items });
+      } else if (subject === "mystery") {
+        this.setState({ mystery: data.items });
+      } else if (subject === "nonfiction") {
+        this.setState({ nonfiction: data.items });
+      } else if (subject === "history") {
+        this.setState({ history: data.items });
+      }
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  componentDidMount() {
+    this.getBookData("horror");
+    this.getBookData("romance");
+    this.getBookData("mystery");
+    this.getBookData("nonfiction");
+    this.getBookData("history");
+  }
+
+  MappedHorrorData = () => {
+    const mappedData = this.state.horror.map((book) => {
+      return (
+        <Card>
+          <Link to={`/book/${book.volumeInfo.industryIdentifiers[0].identifier}`}>
+            <div>{book.volumeInfo.title}</div>
+            <div>
+              <img
+                src={`${book.volumeInfo.imageLinks.thumbnail}`}
+                alt={`${book.volumeInfo.title}`}
+              />
+            </div>
+          </Link>
+        </Card>
+      );
+    });
+    return <Container>{mappedData}</Container>;
+  };
+
+  MappedRomanceData = () => {
+    const mappedData = this.state.romance.map((book) => {
+      return (
+        <Card>
+          <Link to={`/book/${book.id}`}>
+            <div>{book.volumeInfo.title}</div>
+            <div>
+              <img
+                src={`${book.volumeInfo.imageLinks.thumbnail}`}
+                alt={`${book.volumeInfo.title}`}
+              />
+            </div>
+          </Link>
+        </Card>
+      );
+    });
+    return <Container>{mappedData}</Container>;
+  };
+
+  MappedMysteryData = () => {
+    const mappedData = this.state.mystery.map((book) => {
+      return (
+        <Card>
+          <Link to={`/book/${book.id}`}>
+            <div>{book.volumeInfo.title}</div>
+            <div>
+              <img
+                src={`${book.volumeInfo.imageLinks.thumbnail}`}
+                alt={`${book.volumeInfo.title}`}
+              />
+            </div>
+          </Link>
+        </Card>
+      );
+    });
+    return <Container>{mappedData}</Container>;
+  };
+
+  MappedNonfictionData = () => {
+    const mappedData = this.state.nonfiction.map((book) => {
+      return (
+        <Card>
+          <Link to={`/book/${book.id}`}>
+            <div>{book.volumeInfo.title}</div>
+            <div>
+              <img
+                src={`${book.volumeInfo.imageLinks.thumbnail}`}
+                alt={`${book.volumeInfo.title}`}
+              />
+            </div>
+          </Link>
+        </Card>
+      );
+    });
+    return <Container>{mappedData}</Container>;
+  };
+
+  MappedHistoryData = () => {
```

Comment:
All these methods shouldn't be written with initial uppercase right?

Reply:
Yes, you are right. I've made the change.